buildSrc/src/main/java/com/google/firebase/gradle/plugins/ReleaseGenerator.kt

```diff
@@ -59,14 +62,14 @@ data class ReleaseReport(
     """
       |# Release Report
       |${
-      changesByLibraryName.entries.joinToString("\n") {
-      """
+        changesByLibraryName.entries.joinToString("\n") {
+            """
```

Comment:
Intentional?

Reply:
No, The formatting here was a bit weird, I've fixed it now.